test/p2p-test.js

```diff
@@ -73,58 +72,6 @@ describe('P2P', function () {
     await node2.close();
   });
 
-  describe('BIP157', function () {
```

Comment:
Why did we remove these tests?

Reply:
The main reason was that when this test was written, we didn't have handleGetCFHeaders and handleGetCFilters, so it worked perfectly. Currently these functions are recursive and thus the test timed out. And these would automatically be handled if the neutrino-tests passed.